README.md

```diff
@@ -154,10 +154,13 @@ Define which browsers that ez-build should do its best to target when producing
 
 ```bash
 $ ez-build --target-browsers "last 3 versions"
+$ ez-build --target-browsers "Firefox >=44,Chrome >=48,IE >=9,last 2 version"
```

Comment:
because looking at documentation, I had no way of finding out what the format is for having multiple targets - [one, two, three] or 'one','two','three' or 'one,two,three' etc. this is a better example